docker-compose.yml

```diff
@@ -159,6 +159,7 @@ services:
     build:
       context: .
     command: python /opt/warehouse/dev/notdatadog.py 0.0.0.0:8125
+    env_file: dev/environment
```

Comment:
I don't think we need to share the entire environment file with this container.

I'd suggest adding

```
    environment:
      METRICS_OUTPUT: "false"
```